WordPress/gutenberg
# Differentiate 'Copy' and 'Copy styles' snackbar confirmation messages

## What?
Fixes https://github.com/WordPress/gutenberg/issues/57254

Differentiates snackbar confirmation messages for "Copy" and "Copy styles" operations.

## Why?
Currently, the snackbar messages for both "Copy" and "Copy styles" operations display the same text, causing confusion for users. This update introduces distinct messages to improve clarity and the user experience.

## How?

- Updated snackbar messages for "Copy" and "Copy styles" actions.
- Implemented conditional logic to display appropriate messages depending on the action performed.

## Testing Instructions

1. Open the post editor and add a block.
2. Use the "Copy" option from the block toolbar and verify the snackbar displays: Copied "%s" to clipboard.
3. Use the "Copy styles" option from the block toolbar and verify the snackbar displays: Styles copied to clipboard.
4. Ensure both actions work correctly using keyboard navigation.

## Video 

https://github.com/user-attachments/assets/6916f4f0-16f2-492b-8a5b-f1770d2e2b85
